src/staff-administrator/resolvers/staff-user-details-resolver.service.ts

```diff
@@ -3,7 +3,7 @@ import { ActivatedRouteSnapshot, Resolve } from '@angular/router';
 import { combineLatest } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { StaffUser } from '../models/staff-user.model';
-import { StaffDataAccessService } from '../services/staff-data-access/staff-data-access.service';
+import { StaffDataAccessService } from '../../staff-administrator/services/staff-data-access/staff-data-access.service';
```

Comment:
Import ordering should be swapped for relative imports, the more localised the imports, the lower down in the list they should be